app/components/map/comparison/SideBySide.tsx

```diff
@@ -0,0 +1,49 @@
+import L from "leaflet";
+import "leaflet-side-by-side";
+import { useEffect } from "react";
+import { useMap } from "react-leaflet";
+
+interface SideBySideProps {
+  leftDate: string;
+  rightDate: string;
+}
+
+export default function SideBySide({ leftDate, rightDate }: SideBySideProps) {
+  const map = useMap();
+
+  useEffect(() => {
+    const leftLayer = L.tileLayer.wms("/wms", {
+      attribution:
+        '&copy; <a href="https://dataspace.copernicus.eu/" target="_blank">Copernicus Data Space Ecosystem</a>',
+      layers: "TRUE_COLOR",
+      // @ts-expect-error Time is valid but not included in the type definition.
+      time: leftDate,
+      transparent: true,
+    });
+
+    const rightLayer = L.tileLayer.wms("/wms", {
+      attribution:
+        '&copy; <a href="https://dataspace.copernicus.eu/" target="_blank">Copernicus Data Space Ecosystem</a>',
+      layers: "TRUE_COLOR",
+      // @ts-expect-error Time is valid but not included in the type definition.
```

Comment:
I'm not sure this is needed anymore? Does the base leaflet library contain types for these as this was required only for the react version to suppress typescript errors.

Reply:
Without it still gives this in the editor: `Object literal may only specify known properties, and 'time' does not exist in type 'WMSOptions'.ts(2353)`